src/support/z_supervisor_named.erl

```diff
@@ -56,7 +56,7 @@
 }).
 
 -record(child, {
-    pid = undefined,  % pid is undefined when child is not running
+    pid = undefined :: pid() | undefined,  % pid is undefined when child is not running
```

Comment:
Line 59 is too long:     pid = undefined :: pid() | undefined,  % pid is undefined when child is not running.